types/events.go

```diff
@@ -80,6 +82,27 @@ type EventDataRoundState struct {
 	RoundState interface{} `json:"-"`
 }
 
+type ValidatorInfo struct {
+	Address Address `json:"address"`
+	Index   int     `json:"index"`
+}
+
+type EventDataNewRound struct {
+	Height int64  `json:"height"`
+	Round  int    `json:"round"`
+	Step   string `json:"step"`
+
+	Proposer ValidatorInfo `json:"proposer"`
+}
+
+type EventDataCompleteProposal struct {
+	Height int64  `json:"height"`
+	Round  int    `json:"round"`
+	Step   string `json:"step"`
+
+	BlockID         BlockID `json:"block_id"`
```

Comment:
Do we want to add the ValidatorInfo to this event as well? Seems it would make sense

Reply:
I had it there at first, but `EventDataNewRound` seems like the *correct* place to put it. Can add it back if you like.

Reply:
No worries. Let's leave it as is for now - can always add it later.